### ISSUE TITLE
Change to 3 color scheme

### ISSUE THREAD

**Issue by crisslion** [crisslion/utilitarian-css#1]
Have to change the color scheme from 2 color blue/white to light gray/white/black to reduce eye strain from color contrasts

Colors:
Text: #333
BG: #FFF
FG: #EEE;
